pkg/spanconfig/spanconfigkvsubscriber/datadriven_test.go

```diff
@@ -46,8 +46,9 @@ import (
 //      delete [c,e)
 //      upsert [c,d):C
 //      upsert [d,e):D
+//      upsert {entire-keyspace}:_CLUSTER
```

Comment:
Why the _? Drop the cluster verbiage. The text here is just a shorthand for a config, using any letter in the alphabet instead would avoid confusing readers that these words/underscore have special meaning. Mind doing that?